packages/core/src/App/Containers/Layout/header/dashboard-header-dropdown.jsx

```diff
@@ -0,0 +1,125 @@
+                icon: <OurLeadership />,
+            },
+            {
+                name: 'Regulatory information',
+                to: `${deriv_url}/regulatory/`,
+                icon: <RegulatoryInfo />,
+            },
+            {
+                name: 'Why choose us?',
+                to: `${deriv_url}/why-choose-us/`,
+                icon: <WhyChooseUs />,
+            },
+            {
+                name: 'Partnership programmes',
+                to: `${deriv_url}/partners/`,
+                icon: <PartnershipProgrammes />,
+            },
+            {
+                name: 'Contact us',
+                to: `${deriv_url}/contact_us/`,
+                icon: <ContactUs />,
+            },
+            {
+                name: 'Careers',
+                to: `${deriv_url}/careers/`,
+                icon: <Career />,
+            },
+        ];
+    }
+    if (type === 'resources') {
+        return [
+            {
+                name: 'Help centre',
+                to: `${deriv_url}/help-centre/`,
+                icon: <HelpCentre />,
+            },
+            {
+                name: 'Community',
+                to: `${deriv_url}/help-centre/`,
+                icon: <Community />,
+            },
+            {
+                name: 'Payment methods',
+                to: `${deriv_url}/payment-methods/`,
+                icon: <PaymentMethods />,
+            },
+            {
+                name: 'Blog',
+                to: `${deriv_url}/help-centre/`,
+                icon: <Blog />,
+            },
+        ];
+    }
+    return [];
+};
+
+const HeaderDropdownItem = ({ icon, name, to }) => {
+    return (
+        <a href={to} className='dashboard-header-dropdown__card-link'>
+            <div className='dashboard-header-dropdown__card-link__icon'>{icon}</div>
+            <Text size='s'>{name}</Text>
+        </a>
+    );
+};
+
+const HeaderDropdown = ({ current_ref, onClickHandler, parent, setRef }) => {
+    const [left_offset, setLeftOffset] = React.useState(current_ref?.offsetLeft);
+    const dropdown_container_ref = React.useRef(null);
+
+    const updateOffsets = () => {
+        if (current_ref) {
+            setLeftOffset(current_ref.offsetLeft);
+        }
+    };
+
+    React.useEffect(() => {
+        if (dropdown_container_ref) {
+            setRef(dropdown_container_ref);
+        }
+        window.addEventListener('resize', updateOffsets);
+        return () => {
+            window.removeEventListener('resize', updateOffsets);
+        };
+    }, [parent]);
+
+    return (
+        <div ref={setRef} className='dashboard-header-dropdown' style={{ left: left_offset }}>
+            <div className='dashboard-header-dropdown__wrapper'>
+                <div className='dashboard-header-dropdown__wrapper__card'>
+                    {getNavigationItems(parent).map((item, idx) => (
+                        <HeaderDropdownItem key={idx} onClick={onClickHandler} {...item} />
```

Comment:
Since each item's `name` and `to` properties are unique you can use those as the `key` prop rather than `idx`. In general try to avoid using indexes as keys unless you have absolutely no other option.

- https://kentcdodds.com/blog/understanding-reacts-key-prop/
- https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318